gqlalchemy/query_builder.py

```diff
@@ -468,44 +468,64 @@ def from_(
 
     def where(self, item: str, operator: str, value: Any) -> "DeclarativeBase":
         """Creates a WHERE statement Cypher partial query."""
-        separator = "" if operator == ":" else " "
+
+        if operator == ":":
+            separator = ""
+            cypher_value = value
+        else:
+            separator = " "
+            cypher_value = to_cypher_value(value)
+
```

Comment:
You repeat this 3 times, it makes sense to make some auxiliary methods then
it's fine if it's something like `_get_separator` or similar.